src/vs/editor/common/model.ts

```diff
@@ -218,6 +218,10 @@ export interface IModelDecorationOptions {
 	 * with the specified {@link IModelDecorationGlyphMarginOptions} in the glyph margin.
 	 */
 	glyphMargin?: IModelDecorationGlyphMarginOptions | null;
+	/**
+	 * If set, the decoration will override the line height of the lines it spans. This can only increase the line height, not decrease it.
+	 */
+	lineHeight?: number | null;
```

Comment:
qq: why can the line height only be increased?

Reply:
That is legacy text from the original PR, I will remove that.